data/sles4sap/qe_sap_deployment/sles4sap_azure_generic.yaml

```diff
@@ -18,6 +18,7 @@ terraform:
     public_key : '~/.ssh/id_rsa.pub'
     private_key: '~/.ssh/id_rsa'
     sles4sap_uri: 'https://%STORAGE_ACCOUNT_NAME%.blob.core.windows.net/sle-images/%SLE_IMAGE%'
+    os_image: "SUSE:sles-sap-15-sp3:gen2:latest"
     hana_os_major_version: '%HANA_OS_MAJOR_VERSION%'
```

Comment:
How it works with `os_image` hardcoded and `hana_os_major_version` under openQA test control?

Reply:
Tests are still using uploaded image. "os_image" variable is defined statically only so the deployment won't failed and is ignored since "sles4sap_uri" is defined.
Public/private images is part of different task.